en/developer_guide/bundle_configuration.rst

```diff
@@ -66,6 +66,58 @@ This bundle will be used as a base for all the applications api, you will need t
         http_exception_controller:  'OpenOrchestra\BaseApiBundle\Controller\ExceptionController::showAction'
         token_expiration_time:  +1month
 
+ApiBundle
+~~~~~~~~~
+
+This bundle will be used in the applications api. You can configure :
+
+* The ``Facade`` used by each ``Transformer``
```

Comment:
Maybe you can cut off the transformer notion. You could for instance say ``The Facade returned by tha API``